src/components/SearchInput.tsx

```diff
@@ -17,8 +16,8 @@ export const SearchInput = ({ className, onChange, placeholder }: SearchInputPro
   const [localValue, setLocalValue] = useState<string>('')
 
   return (
-    <TextInputForSearch
-      className={className}
+    <TextInput
+      className={tw('max-w-60 [&_input]:h-10 [&_input]:!pl-3', className)}
```

Comment:
Those custom `inputAdornedStart` style sounds like they could be the same for all usage WDYT? Maybe defining them in the MUI default them should do the job?

Reply:
Do you mean a custom class in the muiTheme?

Reply:
More like a style defined in muiTheme for `MuiInputBase-inputAdornedStart`

We already have one defined under the `MuiOutlinedInput` key and it's probably the exact same place you could update 🤔

Reply:
used only for search